src/cyw43_ctrl.c

```diff
@@ -65,6 +69,10 @@ cyw43_t cyw43_state;
 void (*cyw43_poll)(void);
 uint32_t cyw43_sleep;
 
+#if CYW43_ENABLE_BLUETOOTH
+void cyw43_hci_process_func(void);
```

Comment:
I think this should be called `cyw43_bluetooth_hci_process()` and declared alongside the other `cyw43_bluetooth_hci_xxx` functions.  At the very least it should go in a header somewhere.